extension/constants.js

```diff
@@ -90,7 +94,9 @@ constants.SurveyLocation = {
   PHISHING_PROCEED: 'phishing-proceed.js',
   PHISHING_NOPROCEED: 'phishing-noproceed.js',
   EXTENSION_PROCEED: 'extension-proceed.js',
-  EXTENSION_NOPROCEED: 'extension-noproceed.js'
+  EXTENSION_NOPROCEED: 'extension-noproceed.js',
+  HTTPS: 'httpwithans.js',
```

Comment:
I'm not sure I understand what the file names mean here and the next line. Who is Hans and why is he over HTTP?